docs/CHANGELOG.rst

```diff
@@ -18,6 +18,12 @@ Added
 Changed
 -------
 - Add parameter ``--use-original-qualities`` to ``bqsr`` process
+- Update Python package versions in ``resolwebio/rnaseq:6.0.0`` Docker
```

Comment:
All  changes to ``resolwebio/rnaseq:6.0.0`` could be listed as a single entry to the changelog.